posthog/batch_exports/http.py

```diff
@@ -170,11 +170,20 @@ def create(self, validated_data: dict) -> BatchExport:
         destination_data = validated_data.pop("destination")
         team_id = self.context["team_id"]
 
-        if validated_data["interval"] not in ("hour", "day", "week") and not posthoganalytics.feature_enabled(
-            "high-frequency-batch-exports",
```

Comment:
hey @tomasfarias you can try this locally to check if it works or not. (just create the same feature flag in your local env with the same match by and check if it works)

In this case, the feature flag is based on the `project` group, and not distinct IDs, which means you need to pass in the `groups` parameter with key `project` and value as the project ID.

So, you just need to add `project` instead of `organization` in this case, and the property 'id' in `group_properties`

Reply:
One potential gotcha, and reason _not_ to use project and use organisations instead is that currently this will enable it for both, team 2 and 9291 in EU and US both, vs. just those teams in US.

Hence, using org IDs, which are unique across deployments works better usually.

Reply:
We can switch the flag to organizations. That means no code changes are required here, right? Thank you for your input!

Reply:
I've tested things out locally using this branch and flags based on organization and things work 👍 (Discovered a frontend bug while I was at it).